api/src/main/java/org/jboss/forge/roaster/model/Type.java

```diff
@@ -34,6 +38,8 @@
 
    boolean isQualified();
 
+   boolean isNameQualified();
```

Comment:
Ii was thinking from the API consumer perspective. Which method should he/she call if they need to check if a type is qualified?